protocol/app/upgrades.go

```diff
@@ -32,6 +33,17 @@ func (app *App) setupUpgradeHandlers() {
 			app.RatelimitKeeper,
 		),
 	)
+
```

Comment:
Instead for replacing `v4_0_0`, I think we should register additional upgrade handlers here